app/views/confirmation.scala.html

```diff
@@ -0,0 +1,56 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+
+@(appConfig: FrontendAppConfig)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("confirmation.title"),
+    appConfig = appConfig,
+    bodyClasses = None) {
+
+	  <div class="govuk-box-highlight">
+			@components.heading("confirmation.heading", "heading-xlarge")
+      <p class="font-large">
+    		@messages("confirmation.yourRefNumber")<br>
+        <span class="bold">GBJ2123F</span>
```

Comment:
Does this not need to come from the back end?

Reply:
If its not part of the scope of this story thats fine 👍

Reply:
From the acceptance criteria "The submitted application is stored in the database of the operational service for processing"
we can just do this as part of DIT-409

Reply:
"The submitted application is stored in the database of the operational service for processing" That is a different thing. That clearly states "submitted to and stored in the database" NOT to play back generated the ref number to be displayed on the confirmation.